### ISSUE TITLE
As a visitor, I want to have the same layout for a contact information card

### ISSUE THREAD

**Issue by DenMette** [ordina-jworks/ordina-jworks.github.io#476]
Is there a reason why the contact information cards are different on the contact us page and the programs?

Contact Us
![image](https://user-images.githubusercontent.com/849003/50094219-ef5c9600-0212-11e9-9680-ad6093f87251.png)

Kicktstarter and Accelerator
![image](https://user-images.githubusercontent.com/849003/50094298-2f237d80-0213-11e9-9a0c-66bb5bb4f935.png)

**Comment by DenMette** [comment]
I performed a little investigation why the text is aligned left or right.
![image](https://user-images.githubusercontent.com/849003/50140929-80308180-02a6-11e9-8c34-79398978bb7a.png)


Additionally I found the SCSS file corresponding: _wrapper.scss
```css
.wrapper {
  ...

  &.spotlight {
    ...

    &:nth-child(2n - 1) {
      .inner {
        @include vendor('flex-direction', 'row-reverse');
        text-align: right;
      }
      
      .spotlight-image {
        margin: 0 0 _size(element-margin) _size(section-spacing, large);
      }
      
      ...
    } // close spotlight

    ...

} // close wrapper
```

I propose we create a class that just fixed the issue with the contact card information

**Comment by yannickdeturck** [comment]
Fixed in #583.
